app/assets/javascripts/client/inventories/InviteInventoryUserCtrl.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('InviteInventoryUserCtrl', InviteInventoryUserCtrl);
+
+  InviteInventoryUserCtrl.$inject = ['$scope', 'InventoryInvitation'];
+  function InviteInventoryUserCtrl($scope, InventoryInvitation) {
+    var vm = this;
+    vm.sendInvitation = function(invitation) {
+      $scope.alerts  = [];
```

Comment:
This is...strange.  We're defining a field in `$scope` but we're using `vm`?  Also, it's done inside of a method which may never be invoked.  If we attempt to use `alerts` for anything it's undefined on the scope until we invoke `sendInvitation`.